src/sonic-yang-models/yang-models/sonic-device_metadata.yang

```diff
@@ -135,6 +135,10 @@ module sonic-device_metadata {
                     type string;
                 }
 
+                leaf cluster {
+                    type string;
```

Comment:
Please add a description, what does this field contain? when is it used?

Reply:
A cluster is a set of inter-connected computers or hosts. The switches that sits in front belongs to the cluster.
This is a general Networking terminology. -- [Wikipedia](https://en.wikipedia.org/wiki/Computer_cluster)

Description: The switch is a member of this cluster.